code/modules/xenoarchaeology/traits/xenoartifact_minors.dm

```diff
@@ -0,0 +1,430 @@
+// Sharp, makes the artifact do extra damage and slice type
+//============
+/datum/xenoartifact_trait/minor/sharp
+	desc = "Sharp"
+	label_desc = "Sharp: The Artifact is shaped into a fine point. Perfect for popping balloons."
+	blacklist_traits = list(/datum/xenoartifact_trait/minor/dense)
+	flags = PLASMA_TRAIT | URANIUM_TRAIT
+
+/datum/xenoartifact_trait/minor/sharp/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='notice'>The [X.name] feels sharp.</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/sharp/on_init(obj/item/xenoartifact/X)
+	X.sharpness = IS_SHARP_ACCURATE
+	X.force = X.charge_req*0.12
+	X.attack_verb = list("cleaved", "slashed", "stabbed", "sliced", "tore", "ripped", "diced", "cut")
+	X.attack_weight = 2
+	X.armour_penetration = 5
+
+//============
+// Cooler, reduces cooldown times
+//============
+/datum/xenoartifact_trait/minor/cooler
+	desc = "Frosted"
+	label_desc = "Frosted: The Artifact has the unique property of actively cooling itself. This also seems to reduce time between uses."
+	flags = BLUESPACE_TRAIT | PLASMA_TRAIT | URANIUM_TRAIT
+
+/datum/xenoartifact_trait/minor/cooler/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='notice'>The [X.name] feels cold.</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/cooler/on_init(obj/item/xenoartifact/X)
+	X.cooldown = 4 SECONDS //Might revisit the value.
+
+/datum/xenoartifact_trait/minor/cooler/activate(obj/item/xenoartifact/X)
+	X.charge -= 10
+
+//============
+// Sentient, allows a ghost to control the artifact
+//============
+/datum/xenoartifact_trait/minor/sentient
+	label_name = "Sentient"
+	label_desc = "Sentient: The Artifact seems to be alive, influencing events around it. The Artifact wants to return to its master..."
+	///he who lives inside
+	var/mob/living/simple_animal/man
+	///His doorbell
+	var/obj/effect/mob_spawn/sentient_artifact/S
+
+/datum/xenoartifact_trait/minor/sentient/on_touch(obj/item/xenoartifact/X, mob/user)
+	to_chat(user, "<span class='warning'>The [X.name] whispers to you...</span>")
+	return TRUE
+
+/datum/xenoartifact_trait/minor/sentient/on_init(obj/item/xenoartifact/X)
+	addtimer(CALLBACK(src, .proc/get_canidate, X), 5 SECONDS)
+
+/datum/xenoartifact_trait/minor/sentient/proc/get_canidate(obj/item/xenoartifact/X)
+	var/list/mob/dead/observer/candidates = pollGhostCandidates("Do you want to play as the maleviolent force inside the [X.name]?", ROLE_SENTIENCE, null, FALSE, 5 SECONDS, POLL_IGNORE_SENTIENCE_POTION)
+	if(LAZYLEN(candidates))
+		var/mob/dead/observer/C = pick(candidates)
+		setup_sentience(X, C.ckey)
+		return
+	S = new(get_turf(X), X)
+	S.density = FALSE
+
+/datum/xenoartifact_trait/minor/sentient/proc/setup_sentience(obj/item/xenoartifact/X, ckey)
+	if(!(SSzclear.get_free_z_level()))
+		playsound(get_turf(X), 'sound/machines/buzz-sigh.ogg', 50, TRUE) 
+		return	
+	man = new(get_turf(X))
+	man.name = "[pick("Calcifer", "Lucifer", "Ahpuch", "Ahriman")]"
```

Comment:
Take a look out how other things do this, I think soulstone shard gives them no_transform and godmode